studio/styles/event-participant.css

```diff
@@ -0,0 +1,28 @@
+.table-container {
```

Comment:
Noe css, tailwind og sanity spiller ikke bra sammen

Reply:
Satte bare opp tailwind for sveltekit-appen, men kunne evt sett på å sette det opp i studio også. Men vet ikke om det er nødvendig, er kanskje ikke så mye styling som trengs der?

Reply:
Leste noe docs hos sanity som nevnte at det ikke var lett å sette opp i sanity. Er ikke sååå mye styling i studio som vi trenger.

Reply:
Da tenker jeg vi dropper det!